src/neuroconv/datainterfaces/ophys/scanimage/scanimageimaginginterfaces.py

```diff
@@ -535,7 +541,7 @@ def __init__(
         file_path : FilePath
             Path to the TIFF file.
         channel_name : str
-            The name of the channel to load, to determine what channels are available use ScanImageTiffSinglePlaneImagingExtractor.get_available_channels(file_path=...).
+            The name of the channel to load, to determine what channels are available use ScanImageTiffSinglePlaneImagingExtractor.get_avaliable_channels(file_path=...).
```

Comment:
```suggestion
            The name of the channel to load, to determine what channels are available use ScanImageTiffSinglePlaneImagingExtractor.get_available_channels(file_path=...).
```

Reply:
Those changes broke the tests. Don't know if this is an argument for or against testing for exact match with error messages, @pauladkisson.

Reply:
I think it's an argument for. Now we know for sure that the channel and plane error messages render properly.